test/index.js

```diff
@@ -91,14 +103,13 @@ test ('createRunner', () => {
       resultLine,
       /^.*$/,
       /^$/,
-    ])
+    ], run)
   );
 
-  const T = '\u001b\\[90m│\u001b\\[39m';
+  const T = '│';
```

Comment:
This is the only surprising change to me. The table separators seem to no longer be rendered in grey.

Reply:
Ah, I see this change causes the tests to break on CI. So maybe it's the Node version?

Reply:
Nope. The tests pass for me locally on Node 18 too without the expectations of terminal colors. :thinking:

Reply:
No idea.. something on my local system is causing those ansii codes to be suppressed. Oh well. I've amended the commit to keep things how they were.